s3/aide.go

```diff
@@ -94,3 +94,34 @@ func (svc *Service) SetACL(v *string) {
 func (svc *Service) SetSSE(v *string) {
 	svc.sse = v
 }
+
+// ListObjectsKeysV2Pages will list the bucket keys page-wise
+func (svc *Service) ListObjectsKeysV2Pages(params *s3.ListObjectsV2Input) ([]string, bool, error) {
+
+	var keys []string
+	var islastPage bool
```

Comment:
`isLastPage` would follow Go's capitalization standards. `last` or `isLast` would be sufficient, too.

Reply:
Changed it to isLast